src/SEOManager.php

```diff
@@ -166,7 +169,7 @@ public function flipp(string $alias, string|array $data = null): string|static
             ];
         }
 
-        $query = base64_encode(json_encode($data));
+        $query = base64_encode((string) json_encode($data));
```

Comment:
What's the reason for the `(string)`, is it that `json_encode()` can return `false` on fail?

If so, you can pass a second argument with the flag to throw an exception. `JSON_THROW_ON_ERROR`

Reply:
Done.